std_msgs/Header header
PositionBoundingBox2D[] bboxes